lib/fluent/plugin/out_remote_syslog.rb

```diff
@@ -19,6 +19,7 @@ class RemoteSyslogOutput < Fluent::Output
     config_param :facility, :string, :default => "user"
     config_param :severity, :string, :default => "notice"
     config_param :tag, :string, :default => "fluentd"
+    config_param :hostname_key, :string, :default => ""
```

Comment:
not sure how to fix this one @houndci-bot ☝️ 😞  any one that could give me a hand?